cmake/modules/hunter_jobs_number.cmake

```diff
@@ -11,9 +11,14 @@ function(hunter_jobs_number jobs_options_varname toolchain_path)
   string(COMPARE EQUAL "${HUNTER_JOBS_NUMBER}" "" use_default_jobs)
   if(use_default_jobs)
     cmake_host_system_information(
```

Comment:
Actually it would be nice if you can send documentation patch to [CMake](https://gitlab.kitware.com/cmake/cmake).